zorg/jenkins/jobs/jobs/clang-stage1-RA

```diff
@@ -1,6 +1,10 @@
 pipeline {
+    options {
```

Comment:
We should let these do concurrent builds, but throttle on the number of concurrent builds it can do. That would provide a nice balance and keep the changelist short while not using too many nodes.

Reply:
@azharudd I think this would require us to add the [Throttling concurrent builds plugin](https://plugins.jenkins.io/throttle-concurrents/) I believe, so I would propose we do this in a later PR once everything is established. Right now we handle ~10-20 commits on average in our stage1 jobs